samples/core/blur/main.cpp

```diff
@@ -49,6 +49,8 @@ int main(int argc, char* argv[])
         std::tie(use_local_mem, use_subgroup_exchange,
                  use_subgroup_exchange_relative) = blur.query_capabilities();
 
+        bool opencl_c_version_2_0 = blur.query_opencl_c_2_0_support();
```

Comment:
See comment elsewhere regarding checking for OpenCL 2.0 specifically vs. checking for OpenCL 2.0 or newer vs. removing the check completely.